### ISSUE TITLE
password invalid

### ISSUE THREAD

**Issue by CMGS** [influxdb/influxdb#1303]
first you should create a database

```
❯ http -j POST '10.1.201.42:8086/db?u=root&p=root' name=testa                                                                         resoure/git/master !
HTTP/1.1 201 Created
Access-Control-Allow-Headers: Origin, X-Requested-With, Content-Type, Accept
Access-Control-Allow-Methods: GET, POST, PUT, DELETE
Access-Control-Allow-Origin: *
Access-Control-Max-Age: 2592000
Content-Length: 0
Content-Type: text/plain
Date: Thu, 08 Jan 2015 04:55:26 GMT
X-Influxdb-Version: InfluxDB v0.8.4 (git: 7fbb879) (leveldb: 1.15)
```

then, create a database user with password

```
❯ http -j POST '10.1.201.42:8086/db/testa/users?u=root&p=root' name=testa password=12345                                              resoure/git/master !
HTTP/1.1 200 OK
Access-Control-Allow-Headers: Origin, X-Requested-With, Content-Type, Accept
Access-Control-Allow-Methods: GET, POST, PUT, DELETE
Access-Control-Allow-Origin: *
Access-Control-Max-Age: 2592000
Content-Length: 0
Content-Type: text/plain
Date: Thu, 08 Jan 2015 04:56:37 GMT
X-Influxdb-Version: InfluxDB v0.8.4 (git: 7fbb879) (leveldb: 1.15)
```

Now, you will find that you can not login database testa with user testa and password 12345, influxdb will report password invalid, unless you set password again.

```
❯ http -j POST '10.1.201.42:8086/db/testa/users/testa?u=root&p=root' password=12345                                                   resoure/git/master !
HTTP/1.1 200 OK
Access-Control-Allow-Headers: Origin, X-Requested-With, Content-Type, Accept
Access-Control-Allow-Methods: GET, POST, PUT, DELETE
Access-Control-Allow-Origin: *
Access-Control-Max-Age: 2592000
Content-Length: 0
Content-Type: text/plain
Date: Thu, 08 Jan 2015 05:00:09 GMT
X-Influxdb-Version: InfluxDB v0.8.4 (git: 7fbb879) (leveldb: 1.15)
```

now, you can login with password 12345. So why ?

**Comment by toddboom** [comment]
@CMGS Does this also happen with v0.8.8?

**Comment by CMGS** [comment]
@toddboom wow, so fast...I will try 0.8.8 today. Is the latest rpm made by v0.8.8 now?

**Comment by toddboom** [comment]
@CMGS Yes, it should be. All of the packages should be at v0.8.8 now.

If you still find the issue, it's likely that v0.9.0 will solve it since we've done a complete rewrite of all the user authentication and authorization code.

**Comment by CMGS** [comment]
@toddboom  fixed in v0.8.8, hmmm

**Comment by toddboom** [comment]
@CMGS great, thanks for the update!
